### ISSUE TITLE
App not available in Appstore

### ISSUE THREAD

**Issue by rootsh0pf** [mayflower/PayForMe#78]
I am currently in Italy and can't download the app from the Appstore anymore. It says "App not available in your country or region".

Maybe it's the same problem like #65.

**Comment by sargreal** [comment]
Same for me in Germany

**Comment by Hustenbonbon** [comment]
Hello, thanks for the information. 
PayForMe was published to the AppStore by my former employer, Mayflower. Also the initial part of development was done on company time.
I don't do mobile app development anymore and am currently not using PayForMe myself, nor do I have plans to re-publish it in a personal developer accounts. I am going to reach out to Mayflower if there is someone there who wants to fix this issue, but as other issues accumulate while I don't really have time to work on this app, I cannot promise that PayForMe will be available again.

**Comment by rootsh0pf** [comment]
Oh that would be unfortunate! I really enjoyed using your app and I think there is no FOSS alternative at the moment. But I understand the situation and hope someone from Mayflower (or someone else) will take over! :crossed_fingers:

**Comment by InteractionEngineer** [comment]
I might be interested in taking ownership. Haven't looked in the codebase yet but I used this app for two years now and some things always bugged me. Whoever responsible is welcome to contact me. I have an active developer license and experience with Swift and especially SwiftUI.

**Comment by ghost91-** [comment]
@InteractionEngineer that would be great. I really loved using the app and was very surprised to see I could not install it on my new phone… what’s your plan, are you simply going to create a fork?

**Comment by InteractionEngineer** [comment]
@ghost91- No, I would like to talk to the previous owner first to get a better picture of the effort involved and the structure he/she has created to keep this project running for years. If that works out, I want to be a responsible maintainer: It would be the first open source project that I would take on and I will only do that with a proper handover.

**Comment by bgr9** [comment]
Hello, I am interested in also contributing to the evolution of the application without necessarily taking leadership. If an exchange is planned, to present the architecture of the application one to one, I would like to be there. 
I looked at the code and it seems maintainable and scalable.

**Comment by bgr9** [comment]
@ghost91-  @InteractionEngineer 
Hello,
I looked at the code. If we want to do it right, the whole synchronization part with coSpend must be reviewed and updated for iOS, as well as for persistence which uses SQLlite (GRDB) and by getting rid of iHateMoney . We can get some SwiftUI views. So there is a bit of work.
I am up for doing it in the long term. I will give news via a new issue when I have a shell that connects to CoSpend and has a first batch of Views. If others want to contribute, they are welcome.

**Comment by Hustenbonbon** [comment]
Hi all.
Thanks for pinging @bgr9 , I didn't see the new comments before.
I am/was the sole maintainer of PayForMe.

I am very happy to get together and discuss transfering the ownership, giving an overview of the architecture/code etc. I am also happy to occassionaly contribute/fix some problems, but don't have the time anymore to do it on my own.

My email is on my GitHub profile. Everyone who is interested in being part of this discussion, feel free to contact me. @InteractionEngineer I will contact you right now.

**Comment by InteractionEngineer** [comment]
@Hustenbonbon @ghost91- Awesome, let’s get this going! 🙌🏼

**Comment by TheMiNuS** [comment]
Happy to see that the app will continue to exist. I'm using it on a nextcloud server to manage pilots accounts for a flight school. It's super useful. Students appreciate to have live view of their account status.

If some of you manage to recover dev on the app. I see some improvements to do in the future like: 
  - being able to fill the amount with a formula as we can do in the web app. The interest is to be able to calculate directly the price of the lesson as I charge per minutes of flight. For 67m of flight with an hour at X euros I would be able to fill the amount like this (X/60)*67
  - QRCode scan do not work if the nextcloud instance is not at the root of the server.

**Comment by InteractionEngineer** [comment]
Hi @TheMiNuS 
I’m happy to hear you and your students enjoy using PayForMe! If you’d like to suggest a new feature, or report a bug, please create a corresponding issues for that - so that we can keep track of everything.

**Comment by InteractionEngineer** [comment]
On this occasion, @Hustenbonbon and I had a great first meet. I’m optimistic that this issue will be closed soon. 😉
I’ll keep you updated and get you, @bgr9, more closely involved in the next couple of days. 

Looking forward to bring new life to this app!

**Comment by rootsh0pf** [comment]
So cool, thank you for reviving this app! :D
It would be nice if you could also continue to support iHateMoney, since not everyone can self-host Cospend.
But I don't want to ask for too much ;)

**Comment by Zegorax** [comment]
Ciao everyone,

I'm following this issue since my last iOS restore due to the impossibility to install the app again.

I tried again this morning and it worked, apparently it made it back to the AppStore again. https://apps.apple.com/ch/app/payforme/id1500428306?l=fr-FR

**Comment by InteractionEngineer** [comment]
That’s a nice surprise, thanks for sharing! This should be permanent soon. 😉

**Comment by mirrorleos** [comment]
@Hustenbonbon @InteractionEngineer what is the state of the application and/or ownership as of today? I would like to try the app together with Cospend, but if it's not in active development I won't (just because it's for my parents, I don't want to make them change apps too many times as they are not tech savy😂)

**Comment by InteractionEngineer** [comment]
@mirrorleos That's the right approach! 👍🏼
@Hustenbonbon and I had a lively exchange, that I really enjoyed. Unfortunately though, **the ownership and state of development didn't change**. It's not Max's nor my fault, the company behind the repo just doesn't want to act on it. We both contacted Mayflower several times via four different communication channels – but it didn't help. They are unresponsive in this case. 

As I stated right from the beginning, although I am interested in (much) improving the app and keep it alive, I'm only motivated to do so if every active user can benefit from it. I don't want to build a duplicate that primarily confuses people looking for an iOS Cospend client and doesn't help anyone staying with the deprecated one.

**Comment by sargreal** [comment]
@InteractionEngineer this is very sad to hear :/

However, if Mayflower does not respond I do not think it makes sense to waste your enthusiasm about this app!

Forking is a very common thing in open source software, and anyone interested in a cospend client or new features in this app should be able to discover it. If at some point Mayflower decides to change their mind, it should be possible to find a solution of merging the apps again.

P.S. I am also very interested in this app, however I am not able to code anything here since I do not own a Mac right now.

**Comment by Hustenbonbon** [comment]
Hi everyone and sorry that I am a bit late to the party, I have been quite occupied with other stuff.

I need to correct some things:
After @InteractionEngineer and my unreplied messages to Mayflower, I stated that there is no contact on another GitHub issue and right the next day someone else from Mayflower contacted me about it and offered support and the willingness to transfer the repository to me (this was probably two weeks ago now).
I was also made aware that I still had maintainer access to both GitHub and Apple Developer the whole time, I don't know anymore why it appeared to me that this wasn't the case.

I currently don't have the time or motivation to maintain this project again, but I am willing to help and answer questions to anyone who wants to pick this up, regardless if this will be someone from inside Mayflower, @InteractionEngineer or someone else.
